x-pack/plugins/synthetics/server/synthetics_service/formatters/formatting_utils.ts

```diff
@@ -40,6 +47,12 @@ export const replaceStringWithParams = (
   return value as string | null;
 };
 
+export const hasNoParams = (strVal: string) => {
+  const startIndex = strVal.indexOf('${');
+  const endIndex = strVal.indexOf('}');
+  return startIndex === -1 || endIndex === -1 || startIndex > endIndex;
```

Comment:
This will signal a false +ve for a string `} ${abc}`.

```suggestion
  const shellParamsRegex = /\$\{[a-zA-Z_][a-zA-Z0-9_]*\}/g;
  return strVal.match(shellParamsRegex) === null;
```